### DISCUSSION TITLE
Will this work if I try configuring it on a Raspberry Pi 5?

### DISCUSSION THREAD

**Discussion by SecularSteve** [LizardByte/community#406] [Sunshine]
Seems like an ideal potential embedded solution instead of settling for expensive Video Capture Cards to stream to Twitch for example.
I'm settling for this because I want to preserve the performance of the games I'm playing.

But I've never tried either before, so I need some advice.

**Comment by ReenigneArcher** [comment]
https://github.com/LizardByte/Sunshine/pull/2172

[a comment by an automated account is not shown]
